nilearn/connectome/connectivity_matrices.py

```diff
@@ -284,13 +358,17 @@ class ConnectivityMeasure(BaseEstimator, TransformerMixin):
             "covariance", "precision"}, optional
         The matrix kind.
 
+    vectorize : bool, optional
+        Vectorize or not the connectivities.
```

Comment:
Can you write description a bit longer ? May be adding few important lines from function to here.